apps/emqx_gateway/src/coap/README.md

```diff
@@ -300,7 +302,7 @@ CoAP gateway uses some options in query string to conversion between MQTT CoAP.
 
 2.  Session
 
-    Manager the "Transport Mnager" "Observe Resouces Manger" and next message id
+    Manager the "Transport Manager" "Observe Resouces Manger" and next message id
 
 3.  Transport Mnager
```

Comment:
There are other paragraphs with spelling errors, too, so you can double-check them.

```suggestion
3.  Transport Manager
```